sigstore-java/src/main/java/dev/sigstore/oidc/client/GithubActionsOidcClient.java

```diff
@@ -69,6 +75,16 @@ public GithubActionsOidcClient build() {
     }
   }
 
+  @Override
+  public boolean isEnabled() {
+    var githubActions = System.getenv(GITHUB_ACTIONS_KEY);
+    if (githubActions == null || githubActions.isEmpty()) {
+      log.log(Level.INFO, "Github env not detected: skipping github actions oidc");
```

Comment:
Does it qualify for info? Is it more like a debug? (fine?)

Reply:
oh yeah probably `FINE`.